policy/modules/services/tor.if

```diff
@@ -59,3 +59,61 @@ interface(`tor_admin',`
 	files_list_runtime($1)
 	admin_pattern($1, tor_runtime_t)
 ')
+
+#######################################
+## <summary>
+##      Search Tor var library directories.
+## </summary>
+## <param name="domain">
+##      <summary>
+##      Domain allowed access.
+##      </summary>
+## </param>
+#
+interface(`tor_search_var_lib',`
+        gen_require(`
+                type tor_var_lib_t;
+        ')
+
+        files_search_var_lib($1)
+        allow $1 tor_var_lib_t:dir search_dir_perms;
+')
+
+#######################################
+## <summary>
+##      Search Tor pluggable transport state var library directories.
+## </summary>
+## <param name="domain">
+##      <summary>
+##      Domain allowed access.
+##      </summary>
+## </param>
+#
+interface(`tor_pt_state_search_var_lib',`
+        gen_require(`
+                type tor_pt_state_var_lib_t;
+        ')
+
+        files_search_var_lib($1)
+        allow $1 tor_pt_state_var_lib_t:dir search_dir_perms;
+')
+
+########################################
+## <summary>
+##      Read and write Tor pluggable transport state var files.
+## </summary>
+## <param name="domain">
+##      <summary>
+##      Domain allowed access.
+##      </summary>
+## </param>
+#
+interface(`tor_pt_state_rw_var_files',`
+        gen_require(`
+                type tor_var_lib_t;
+                type tor_pt_state_var_lib_t;
+        ')
+
+        files_search_var_lib($1)
+        rw_files_pattern($1, tor_var_lib_t, tor_pt_state_var_lib_t)
+')
```

Comment:
Based on the usage in obfs4proxy.te, these can be merged into a single interface.